feature/collection-sheet/build.gradle.kts

```diff
@@ -9,4 +9,11 @@ android {
 
 dependencies {
 
+    implementation(project(":core:datastore"))
```

Comment:
have you tried projects.core.datastore, check it should be accessible

Reply:
I made this change